packages/server/src/workers/download.ts

```diff
@@ -181,7 +181,7 @@ export async function execDownloadJob(job: Job<DownloadJobData>): Promise<void>
     }
 
     const contentDisposition = response.headers.get('content-disposition') as string | undefined;
-    const contentType = response.headers.get('content-type') as string | undefined;
```

Comment:
I'm a little surprised a lint didn't flag this inadvertent bitwise OR

Reply:
Hm, I'm pretty sure this is valid TypeScript, right?

<img width="697" alt="image" src="https://github.com/medplum/medplum/assets/749094/1effd41a-1f08-467c-96ce-e5fdd309cce9">